.gitignore

```diff
@@ -7,3 +7,6 @@
 build
 .DS_STORE
 local.properties
+docs/index.md
+docs/license.md
+site/
```

Comment:
The `site` is where MkDocs outputs the HTML for the website that is used in the `gh-pages` branch.